chain/chain/src/flat_storage_resharder.rs

```diff
@@ -684,7 +708,9 @@ impl FlatStorageResharder {
                 )),
             );
             store_update.commit()?;
+
             num_batches_done += 1;
+            metrics.set_head_height(chain_store.get_block_height(&flat_head_block_hash)?);
```

Comment:
Nice, but doesn't flat storage already have a metric and update logic for head?

Reply:
Yes, but it's not updated during catchup, so without this addition one wouldn't see any progress, just the flat head moving from `resharding_block` to `chain final head` once catchup is finished